terraform/gcp/single-node/main.tf

```diff
@@ -0,0 +1,85 @@
+resource "google_compute_network" "airbyte-network" {
```

Comment:
I think we should only focus on creating the resources that are related to the instance, not network nor subnetworks, nor NAT. These should be specified as variables so you can hook your environement